ts/Accessibility/Components/SeriesComponent/SeriesDescriber.ts

```diff
@@ -613,13 +616,24 @@ function defaultSeriesDescriptionFormatter(
         ) || chart.langFormat(
             'accessibility.series.summary.default' + combinationSuffix,
             summaryContext
-        );
-
-    return summary + (description ? ' ' + description : '') + (
-        shouldDescribeAxis('yAxis') ? ' ' + yAxisInfo : ''
-    ) + (
-        shouldDescribeAxis('xAxis') ? ' ' + xAxisInfo : ''
-    );
+        ),
+        axisDescription = (
+            shouldDescribeAxis('yAxis') ? ' ' + yAxisInfo + '.' : ''
+        ) + (
+            shouldDescribeAxis('xAxis') ? ' ' + xAxisInfo + '.' : ''
+        ),
+        formatStr = chart.options.accessibility.series.descriptionFormat || '';
+
+    return format(formatStr, {
+        seriesDescription: summary,
+        authorDescription: (description ? ' ' + description : ''),
+        axisDescription,
+        series,
+        numSeries,
+        numPoints,
+        chart,
+        index
```

Comment:
Just thinking - do we need all of these shorthands? Or could for instance `{numSeries}` as easily be written `{chart.series.length}` if we add `chart` to the context and show this in docs and demos`. The demos previously, when using formatters, used more of those paths that correspond to the class reference.

Reply:
I see what you mean, although since number of series and points are very common to add to these format strings I figured it would help the users if we gave them shorthands. Happy to remove them if you think it's too bloaty though?

Reply:
Yes I am leaning towards removing them and adding `chart` instead. Because `{chart.series.length}` is not more complicated to use than `{numSeries}`, provided that they are documented the same way. But `chart.series.length` additionally reflects the actual structure in Highcharts.

Reply:
Hm, looking at this further our fancy formatting functions like `#plural()` won't currently work with property access. I can try to add it, but could get a bit convoluted.

Context: `line: '{name}, line {seriesNumber} of {chart.series.length} with {series.points.length} data {#plural(series.points.length, points, point)}.'`

Reply:
Well, we already have `getNestedProperty` separated out in utils, so hopefully not too convoluted then - I'll give it a shot